tests/test_aea.py

```diff
@@ -167,7 +167,7 @@ def test_react():
             t.start()
             time.sleep(2.0)
             dummy_skill = agent.resources.get_skill("dummy")
-            dummy_handler = dummy_skill.handlers[0]
+            dummy_handler = dummy_skill.handlers["dummy"]
```

Comment:
maybe use the new method: `dummy_skill.handlers.dummy` as well and add an assert so we show both are possible